src/hotel/src/service/hotelService.js

```diff
@@ -0,0 +1,43 @@
+var hotels = require(process.env.INIT_CWD +"/hotel-data.json");
+var hotelInfo = require(process.env.INIT_CWD +"/hotel-info.json");
+
+function getHotels(city, country, f) {
+  country = country
+    .trim()
+    .toLowerCase()
+    .replace("%20", " ")
+    .replace(/\w\S*/g, (w) => (w.replace(/^\w/, (c) => c.toUpperCase())));
+  city = city    
+    .trim()
+    .toLowerCase()
+    .replace("%20", " ")
+    .replace(/\w\S*/g, (w) => (w.replace(/^\w/, (c) => c.toUpperCase())));
+  var data = hotels[country][city];
+  if(!f && data){
+    return data;
+  }
+  else if(data){
+    data = data.filter((hotel) => {
+      var matchSuperchain = f.superchain.length === 0 || f.superchain.includes(hotel.superchain);
+      var matchHotel = f.hotel.length === 0 || f.hotel.includes(hotel.name);
+      var matchType = f.type.length === 0 || f.type.includes(hotel.type);
+      var costHigherThan = hotel["cost"] > f.minCost;
+      var costLowerThan = hotel["cost"] < f.maxCost;
+      return matchSuperchain && matchHotel && matchType && costHigherThan && costLowerThan;
+    });
+  }
+
+  return data;
+}
+
+
+
+function getHotelWithFilter() {
```

Comment:
Is this going to be used later?

Reply:
i managed to get it working without using the extra filter. 😂 
will remove.